### PR TITLE
feat/referenda vote distribution and tips

### PR DESCRIPTION
**Pull request by This-Is-Prince**
### Showing tips using new api and it is also backward compatible

Screenshot 2022-10-20 at 12 32 13 AM

### CODE REVIEW

**Comment by This-Is-Prince** [comment]
### Started

Screenshot 2022-10-20 at 10 06 09 PM

### Passed

Screenshot 2022-10-20 at 10 06 34 PM

File: front-end/src/components/Post/GovernanceSideBar/index.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ interface Props {
 	startTime: string
 }
 
-const GovenanceSideBar = ({ canEdit, className, isMotion, isProposal, isReferendum, isTipProposal, isTreasuryProposal, onchainId, onchainLink, startTime, status }: Props) => {
+const GovernanceSideBar = ({ canEdit, className, isMotion, isProposal, isReferendum, isTipProposal, isTreasuryProposal, onchainId, onchainLink, startTime, status }: Props) => {
```
Comment (alphainfinitus) [inline front-end/src/components/Post/GovernanceSideBar/index.tsx:46]: This is the default and only export so it wouldn't break things. But please fix the typo in the file references too.

**Review by alphainfinitus — CHANGES_REQUESTED** [review]
Please fix the 'GovernanceSideBar' typo in the file references too.

File: front-end/src/components/Post/GovernanceSideBar/index.tsx (hunk 1 above)
Comment (This-Is-Prince) [reply front-end/src/components/Post/GovernanceSideBar/index.tsx:46]: Done

**Review by alphainfinitus — APPROVED** [review]

**Pull request merged**